client/python/pyproject.toml

```
@@ -4,7 +4,7 @@ version = "0.2.1"
 description = "Armada gRPC API python client"
 readme = "README.md"
 requires-python = ">=3.7,<3.9"
-dependencies = ["grpcio~=1.46.3", "grpcio-tools~=1.46.3", "mypy-protobuf>=3.2.0"]
+dependencies = ["grpcio~=1.49", "grpcio-tools~=1.49", "mypy-protobuf>=3.2.0"]
```

@jayofdoom has suggested that we maybe not so strictly pinn our dependencies.  

```suggestion
dependencies = ["grpcio>=1.46.3", "grpcio-tools>=1.46.3", "mypy-protobuf>=3.2.0"]
```

If this is backwards compatible.  If not, then we can bump up the version and have it be greater than equal too.